Library/Homebrew/cmd/postinstall.rb

```diff
@@ -2,7 +2,7 @@
 
 module Homebrew
   def postinstall
-    ARGV.resolved_formulae.each { |f| run_post_install(f) }
+    ARGV.resolved_formulae.select(&:post_install_defined?).each { |f| run_post_install(f) }
```

Comment:
Yes, I think we can discard this as this is hardly a hotspot which requires optimization. But thanks for your comment anyway. :smile: